src/daemon/ta/proxy.rs

```diff
@@ -186,6 +219,9 @@ impl fmt::Display for TrustAnchorProxyCommandDetails {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         // note that this is a summary, full details are stored in the json.
         match self {
+            TrustAnchorProxyCommandDetails::Initialise => {
+                write!(f, "Create TA proxy")
```

Comment:
Same comment as elsewhere as to whether this should be "create" or "initialise".